bdb/column.go

```diff
@@ -29,6 +29,11 @@ type Column struct {
 	// tinyint(1) instead of tinyint
 	// Used for "tinyint-as-bool" flag
 	FullDBType string
+
+	// MS SQL only bits
+	// Used to indicate that the value
+	// for this column is auto generated by database on insert (i.e. - timestamp (old) or rowversion (new))
+	AutoGenerated bool
```

Comment:
Also note that postgres/mysql implement auto-generated columns as a sort of complex default.